azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/ActionBarView.kt

```diff
@@ -47,23 +54,25 @@ internal fun ActionBarView(
             backgroundColor = Color.White,
             title = {
                 Column(
-                    modifier = Modifier.padding(start = 90.dp),
+                    modifier = Modifier
+                        .padding(start = 90.dp)
+                        .clickable {
+                            if (!isShowingParticipants) {
+                                postAction(
+                                    ParticipantAction.ParticipantsListVisibilityChanged(visible = true)
+                                )
```

Comment:
Would it be more correct to dispatch a navigation action to go to the participant list instead?